CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## [1.2.31](https://github.com/ably/ably-java/tree/v1.2.31)
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.30...v1.2.31)
+
+**Fixed bugs:**
+
+- Error code for channel attachment timed out [\#959](https://github.com/ably/ably-java/issues/959)
+- Error code for message decoding failure [\#958](https://github.com/ably/ably-java/issues/958)
+- Fix incremental backoff while reconnecting [\#954](https://github.com/ably/ably-java/issues/954)
+- Add missing clientOptions [\#956](https://github.com/ably/ably-java/issues/956)
```

Comment:
tbh i think all of these issue titles are quite vague/non-obvious, could we editorialise them a bit?
```suggestion
- Update error code for channel attachment timed out [\#959](https://github.com/ably/ably-java/issues/959)
- Update error code for message decoding failure [\#958](https://github.com/ably/ably-java/issues/958)
- Fix incremental backoff while reconnecting [\#954](https://github.com/ably/ably-java/issues/954)
- Add `suspendedRetryTimeout` and `httpMaxRetryDuration`  [\#956](https://github.com/ably/ably-java/issues/956)
```